### ISSUE TITLE
GLSL shaders challenges

### ISSUE THREAD

**Issue by olliraa** [joncampbell123/dosbox-x#1926]
For some reason I cannot get some of GLSL shaders that work e.g. some other Dosbox ports. Either the shader does nothing or the screen remains completely black. This could be of course due to misconfiguration, but like I said the same config works with another port. DOS Box-X is the best of all the ports, so this would be the icing on the cak
[GLSL_shaders_with_problems.zip](https://github.com/joncampbell123/dosbox-x/files/5325377/GLSL_shaders_with_problems.zip)

e :) 

I've attached 3 problematic GLSL files. Could someone test hem, if they work for you. Here's the specs:

DOSBox-x 0.83.6 (tested with SDL1/SDL2/32b/64b)
Windows 10
Latest Nvidia drivers (GTX1080Ti)

**Comment by Wengier** [comment]
I have tested the 3 GLSL shaders in the zip file, namely:

* 1crt-guest-dr-venom-pass2.glsl
* crt-guest-dr-venom-pass2.glsl
* crt-guest-dr-venom.glsl

The first shader seems to give black screen in DOSBox-X as well as DOSBox SVN and DOSBox ECE when I tested it. And the other two shaders seem to have errors in themselves and will trigger the following messages when I tested it in DOSBox SVN:

![image](https://user-images.githubusercontent.com/8216923/95045264-92fc8c00-06af-11eb-83c4-848c0eb2bdb5.png)

So according to my tests the issues with these 3 shaders do not only occur in DOSBox-X. Please there is something wrong with them?

**Comment by olliraa** [comment]
> I have tested the 3 GLSL shaders in the zip file, namely:
> 
> * 1crt-guest-dr-venom-pass2.glsl
> * crt-guest-dr-venom-pass2.glsl
> * crt-guest-dr-venom.glsl
> 
> The first shader seems to give black screen in DOSBox-X as well as DOSBox SVN and DOSBox ECE when I tested it. And the other two shaders seem to have errors in themselves and will trigger the following messages when I tested it in DOSBox SVN:
> 
> ![image](https://user-images.githubusercontent.com/8216923/95045264-92fc8c00-06af-11eb-83c4-848c0eb2bdb5.png)
> 
> So according to my tests the issues with these 3 shaders do not only occur in DOSBox-X. Please there is something wrong with them?

You're correct. Due to my misconfiguration, the other DOSBox versions apparenly actually used crt-lottes. Unfortunately, I was foolish enough to think that I was actually using one of the ones I sent in the zip. My bad, sorry :( 

Having the crt-royale would still me very nice, if possible at some point :)

DOSBox-X is a really smooth experience. Kudos!
